Earthfile

```diff
@@ -146,6 +146,16 @@ earth-docker:
     COPY --build-arg VERSION=$TAG +earth/earth /usr/bin/earth
     SAVE IMAGE --push earthly/earth:$TAG
 
+# we abuse docker here to distribute our binaries
+prerelease-docker:
+    ARG EARTHLY_GIT_HASH
```

Comment:
This is not necessary, right?

Reply:
good catch, I used to be using it in the tag, but no longer am and will remove it.